ansible/roles/stack-sunbird/templates/analytics_api_service.conf

```diff
@@ -226,7 +226,8 @@ deviceRegisterAPI.experiment.enable=false
 experimentService.redisEmptyValueExpirySeconds=86400
 redis.experimentIndex=9
 redis.deviceIndex=2
-druid.coordinator.host="http://{{groups['raw-coordinator'][0]}}:8081/"
+#druid.coordinator.host="http://{{groups['raw-coordinator'][0]}}:8081/"
+druid.coordinator.host="{{ druid_coordinator_host | default(http://{{groups['raw-coordinator'][0]}}:8081/)"
```

Comment:
@sowmya-dixit Please change this as below
```
"{{ druid_coordinator_host | default('http://' + groups['raw-coordinator'][0] + ':8081/') }}"
```

Reply:
Done